sdk/resourcemanager/azure-resourcemanager-containerinstance/pom.xml

```diff
@@ -48,6 +48,7 @@
       --add-opens com.azure.core/com.azure.core.implementation.util=ALL-UNNAMED
     </javaModulesSurefireArgLine>
     <doclintMissingInclusion>-</doclintMissingInclusion>
+    <revapi.skip>true</revapi.skip>
```

Comment:
why?

Reply:
Changed the return value of `withDnsPrefix` to `WithDnsNameLabelReusePolicy` is a breaking change. I tried adding ignore items in `revapi.json` but it didn't seem to work. So I had to add `<revapi.skip>true</revapi.skip>` in `pom.xml`.

Reply:
Nothing should break in this kind of feature PR.

Reply:
Fixed in the new version.